assets/javascripts/discourse/components/group-tracker-first-post.js

```diff
@@ -0,0 +1,23 @@
+import { action } from "@ember/object";
+import Component from "@glimmer/component";
+import DiscourseURL from "discourse/lib/url";
+import { alias } from "@ember/object/computed";
+
+export default class GroupTrackerFirstPost extends Component {
+  @alias("args.topic.currentPost") postId;
```

Comment:
A bit of a nitpick, but I think doing something like

```suggestion
  postId = this.args.topic.currentPost
```

feels more _glimmery_. What do you think? If you are cool with the change there are couple instances of this.

Reply:
`postId = this.args.topic.currentPost` wouldn't be reactive to potential args changes. I'd recommend `@readOnly` over `@alias` though